package.json

```diff
@@ -25,8 +25,7 @@
     "build:script:projects": "lerna run build:script --scope=@tps/apps-projects",
     "build:script:rewards": "lerna run build:script --scope=@tps/apps-rewards",
     "clean:aragon": "rm -rf ~/.aragon && rm -rf ~/.ipfs",
-    "clean:build": "npm run clean -- --exclude=node_modules/",
-    "clean:commit": "rm -f package-lock.json && npm run bootstrap",
+    "clean:build": "git clean -fxd --exclude=node_modules/",
```

Comment:
Should this use `-fXd`?

Reply:
https://github.com/AutarkLabs/open-enterprise/pull/1264#discussion_r329854509